shared/actions/config/index.js

```diff
@@ -91,7 +93,28 @@ function getMyFollowing (username: string): AsyncAction {
   }
 }
 
-function getExtendedStatus (): AsyncAction {
+export function waitForKBFS (): AsyncAction {
+  return dispatch => {
+    return new Promise((resolve, reject) => {
+      configWaitForClientRpc({
+        param: {clientType: 3, timeout: 10.0},
```

Comment:
:sheep: Is there a constant definition you can use here instead of the magic number `3`?